core/src/main/kotlin/org/evomaster/core/output/naming/LanguageConventionFormatter.kt

```diff
@@ -0,0 +1,35 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.output.OutputFormat
+
+class LanguageConventionFormatter(
```

Comment:
add couple of sentences in JavaDoc to state what this class is for. eg `Different programming languages have different conventions and style when defining variable names and other programming structures. As such, when we generate tests, ideally we would like to follow the same conventions`.

Reply:
Done